static/js/components/search/CustomSortingSelect.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Select } from 'searchkit';
+import block from 'bem-cn';
+
+import type { Event } from '../../flow/eventType';
+
+export default class CustomSortingSelect extends Select {
+  showDropdown(sortSelectField: string, clickEvent: Event): void {
+    clickEvent.preventDefault();
+    let event: Event = document.createEvent('MouseEvents');
+    event.initMouseEvent('mousedown', true, true, window);
+    ReactDOM.findDOMNode(sortSelectField).dispatchEvent(event);
+  }
+
+  optionText(
+    translate: Function,
+    showCount: number,
+    countFormatter: Function,
+    items: Array<Object>
+  ): string {
```

Comment:
this return type isn't accurate anymore :)

also maybe rename the function to just `options` or `formatOptions` or something along those lines?